source/includes/administration/_environments.md

```diff
@@ -167,16 +167,28 @@ curl -X POST "https://cloudmc_endpoint/v2/environments" \
   },
   "serviceConnection": {
     "id": "7f0fa906-490a-467b-bc44-e2382d43015e"
-  },
-  "roles": [{
-    "name": "Environment Admin",
-    "isDefault": true,
-    "users": [{
-      "id": "73b17dab-1705-45f1-84e2-997e2af5641b"
-    }]
-  }]
+  }
 }
 ```
+
+Required | &nbsp;
+-------- | -----------
+`name`<br/>*string* | The name of the new environment. Should be unique in the environment and only contain lower case characters, numbers, dashes and underscores.
+`serviceConnection`<br/>*[ServiceConnection](#administration-service-connections)* | The service connection that the environment should be created in<br/>*required*: `id`
+
+Optional | &nbsp;
+-------- | -----------
+`description`<br/>*string* | The description of the new environment.
+`organization`<br/>*[Organization](#administration-organizations)* | The organization that the environment should be created in. *Defaults to your organization*<br/>*required*: `id`
```

Comment:
should this be moved to required? Or if it's default, maybe remove the *required*? Looks confusing to me

Reply:
the organization is optional - the id is what is required. If you go to the rendered view in github it should be more clear